### PR TITLE
Round2: Generated by scripts

### PR DESCRIPTION
**Pull request by caesarxuchao**
Last round of PRs failed to update vendor/. This one fixed it.

@deads2k @sttts 

PRs to other repos:
https://github.com/kubernetes/apiserver/pull/5
https://github.com/kubernetes/kube-aggregator/pull/5
https://github.com/kubernetes/sample-apiserver/pull/3
https://github.com/kubernetes/client-go/pull/154
https://github.com/kubernetes/apimachinery/pull/11

### CODE REVIEW

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -8,10 +8,12 @@
 	"Deps": [
 		{
 			"ImportPath": "bitbucket.org/ww/goautoneg",
+			"Comment": "null-5",
```
Comment (caesarxuchao) [inline Godeps/Godeps.json]: These comments are added back because the Godeps.json is now generated by `godep save`. I think the comment is innocuous.

Comment (sttts) [inline Godeps/Godeps.json]: As long as our `hack/verify-*` scripts do not fall over them. We should have the ignore flag in the diff commands everywehere.

**Comment by deads2k** [comment]
lgtm.  let me know about merge.

File: Godeps/Godeps.json (hunk 1 above)
Comment (caesarxuchao) [inline Godeps/Godeps.json]: I didn't change the copy in staging. hack/verify doesn't check repos, so we should be ok.

**Comment by deads2k** [comment]
merging for script

**Pull request merged**